common/perception_utils/include/perception_utils/matching.hpp

```diff
@@ -22,46 +22,55 @@
 #include <boost/geometry.hpp>
 
 #include <algorithm>
+#include <numeric>
+#include <utility>
 #include <vector>
 
 namespace perception_utils
 {
-inline double getConvexShapeArea(
-  const tier4_autoware_utils::Polygon2d & source_polygon,
-  const tier4_autoware_utils::Polygon2d & target_polygon)
+using tier4_autoware_utils::Polygon2d;
+
+inline double getConvexShapeArea(const Polygon2d & source_polygon, const Polygon2d & target_polygon)
 {
-  boost::geometry::model::multi_polygon<tier4_autoware_utils::Polygon2d> union_polygons;
+  boost::geometry::model::multi_polygon<Polygon2d> union_polygons;
   boost::geometry::union_(source_polygon, target_polygon, union_polygons);
 
   tier4_autoware_utils::Polygon2d hull;
   boost::geometry::convex_hull(union_polygons, hull);
   return boost::geometry::area(hull);
 }
 
-template <class T1, class T2>
-double get2dIoU(const T1 source_object, const T2 target_object)
+inline double getAreaSum(const std::vector<Polygon2d> & polygons)
```

Comment:
[imo] `getSumArea` may be better to match other functions like `getIntersectionArea` or `getUnionArea`

Reply:
Thank you for polite review.
Fixed at [c313888](https://github.com/autowarefoundation/autoware.universe/pull/2045/commits/c313888f61700cd229b78621cd4854c1ff814197)